linkerd/app/core/Cargo.toml

```diff
@@ -63,7 +63,7 @@ linkerd2-stack-tracing = { path = "../../stack/tracing" }
 linkerd2-trace-context = { path = "../../trace-context" }
 rand = { version = "0.7", features = ["small_rng"] }
 regex = "1.0.0"
-tokio = { version = "0.2", features = ["macros", "sync"]}
+tokio = { version = "0.2", features = ["macros", "sync", "parking_lot"]}
```

Comment:
Do we need to set this is in all places that `sync` is used?

Reply:
Nope --- if one crate enables the feature (and in this case, `linkerd2_app_core` is always in the tree when building the actual or profiling proxies) the feature is enabled globally. This behavior of cargo features is kind of a blessing and a curse. :)

It would only matter if we needed it to be enabled when building individual crates on their own --- in this case, we don't, because those crates will still _compile_ without `parking_lot`, and rust-analyzer is the only real reason we build individual crates separately from the proxy.